### PR TITLE
[Polska] Updated all files

### PR DESCRIPTION
**Pull request by slav123**
RE: #148 update for PL locale

### CODE REVIEW

**Review by paulbalandan — CHANGES_REQUESTED** [review]
We need to pass all the tests  for the Polska locale. Some notes:

- Do not change the indentation of the `=>`. This is automatically fixed by our fixer.
- Supply the missing language strings.
- Remove the extra file, Seed.php.
- __IMPORTANT:__ Have your commits GPG signed for merging to be possible.
- Details of test failures can be found in the test results.
- In the README, please update also the link to your profile. Do not change the release version of pl to `4.x`

File: Language/pl/Session.php [hunk 1]
```diff
@@ -16,4 +16,5 @@
 	'writeProtectedSavePath' => 'Sesja: Skonfigurowana ścieżka zapisu "{0}" nie jest zapisywalna w procesie PHP.',
 	'emptySavePath'          => 'Sesja: Nie skonfigurowano żadnej ścieżki zapisu.',
 	'invalidSavePathFormat'  => 'Session: Nieprawidłowy format zapisu zapisu Redis: {0}',
+	'invalidSameSiteSetting' => 'Session: The SameSite setting must be None, Lax, Strict, or a blank string. Given: {0}',
```
Comment (paulbalandan) [inline Language/pl/Session.php:19]: No translation for this?

File: Language/pl/HTTP.php [hunk 2]
```diff
@@ -68,5 +68,8 @@
 	'uploadErrCantWrite' => 'Plik "%s" nie może być zapisany na dysku.',
 	'uploadErrNoTmpDir'  => 'Plik nie może zostać wysłany. Brak katalogu tymczasowego.',
 	'uploadErrExtension' => 'Wysyłka pliku została zablokowana przez PHP.',
-	'uploadErrUnknown'   => 'Plik "%s" nie został wysłany z nieznanego powodu.'
+	'uploadErrUnknown'   => 'Plik "%s" nie został wysłany z nieznanego powodu.',
+
+	// SameSite setting
+	'invalidSameSiteSetting'     => 'The SameSite setting must be None, Lax, Strict, or a blank string. Given: {0}',
```
Comment (paulbalandan) [inline Language/pl/HTTP.php:74]: No translation?

File: Language/pl/Database.php [hunk 3]
```diff
@@ -15,15 +15,16 @@
...
-	'failGetFieldData'                 => 'Failed to get field data from database.',
-	'failGetIndexData'                 => 'Failed to get index data from database.',
-	'failGetForeignKeyData'            => 'Failed to get foreign key data from database.',
-	'parseStringFail'                  => 'Parsing key string failed.',
-	'featureUnavailable'               => 'This feature is not available for the database you are using.',
-	'tableNotFound'                    => 'Table `{0}` was not found in the current database.',
-	'noPrimaryKey'                     => '`{0}` model class does not specify a Primary Key.',
-	'noDateFormat'                     => '`{0}` model class does not have a valid dateFormat.',
-	'fieldNotExists'                   => 'Field `{0}` not found.',
+	'emptyPrimaryKey'                  => 'There is no primary key defined when trying to make {0}.',
```
Comment (paulbalandan) [inline Language/pl/Database.php:18]: No translation?

File: Language/pl/Fabricator.php [hunk 4]
```diff
@@ -11,6 +11,7 @@
 
 // Fabricator language strings
 return [
-	'invalidModel'      => 'Invalid model supplied for fabrication.',
-	'missingFormatters' => 'No valid formatters defined.',
+	'invalidModel'      => 'Niepoprawny model do fabrykacji.',
+	'missingFormatters' => 'Brak poprawnego formatu.',
+	'createFailed'      => 'Fabricator failed to insert on table {0}: {1}.',
```
Comment (paulbalandan) [inline Language/pl/Fabricator.php:16]: No translation?

File: Language/pl/Format.php [hunk 5]
```diff
@@ -11,6 +11,8 @@
 
 // Format language settings
 return [
+	'invalidFormatter' => '"{0}" is not a valid Formatter class.',
```
Comment (paulbalandan) [inline Language/pl/Format.php:14]: No translation?

File: Language/pl/Format.php [hunk 6]
```diff
@@ -11,6 +11,8 @@
...
 	'invalidJSON'      => 'Nie można przeanalizować łańcucha JSON, błąd: "{0}".',
+	'invalidMime'      => 'No Formatter defined for mime type: "{0}".',
```
Comment (paulbalandan) [inline Language/pl/Format.php:16]: No translation?

File: Language/pl/Migrations.php [hunk 7]
```diff
@@ -30,25 +30,28 @@
...
+	'generalFault'      => 'Migracja nie powiodła się!',
 	'migInvalidVersion' => 'Podano nieprawidłowy numer wersji.',
 	'toVersionPH'       => 'Migrowanie do wersji %s...',
 	'toVersion'         => 'Migracja do bieżącej wersji...',
 	'rollingBack'       => 'Wycofuję wszystkie migracje...',
 	'noneFound'         => 'Nie znaleziono migracji.',
-	'on'                => 'Migracja włączona: ',
 	'migSeeder'         => 'Nazwa Seedera',
 	'migMissingSeeder'  => 'Musisz podać nazwę seedera.',
+	'nameSeeder'        => 'Name the seeder file',
```
Comment (paulbalandan) [inline Language/pl/Migrations.php:46]: No translation?

File: Language/pl/Migrations.php [hunk 8]
```diff
@@ -30,25 +30,28 @@
...
 	'added'             => 'Uruchomienie: ',
 
-	'version'  => 'Wersja',
+	// Migrate Status
+	'namespace'         => 'Namespace',
 	'filename' => 'Nazwa pliku',
+	'version'  => 'Wersja',
+	'group'             => 'Group',
+	'on'                => 'Migrated On: ',
+	'batch'             => 'Batch',
```
Comment (paulbalandan) [inline Language/pl/Migrations.php:56]: No translations?

**Review by paulbalandan — COMMENTED** [review]
See comments below for missing translations.

Your first two commits are not signed so I am forbidden to merge this PR just in case I approve this. This repo requires all commits to be signed before merge. If you can sign those commits along with my comments below that would be great.

**Comment by slav123** [comment]
I signed last commits, translation isn't completed because my time for opensource projects is limited. I finish them when I do have some spare time to work on it.

**Pull request closed without merging**